app/components/forms/wizard/basic-details-step.js

```diff
@@ -121,6 +121,13 @@ export default Component.extend(FormMixin, EventWizardMixin, {
   },
 
   getValidationRules() {
+    window.$.fn.form.settings.rules.checkMaxMin = () => {
+      let returnValue = false;
+      if (this.$('.ui.form').form('get value', 'min_price')[0] < this.$('.ui.form').form('get value', 'max_price')[0]) {
```

Comment:
@mrsaicharan1 could you explain the reasoning behind this ?

Reply:
This is just to validate that the min_price is lesser than max_price. `this.$('.ui.form').form('get value', 'min_price')` gave me access to the array object where the first element was the value.

Reply:
@mrsaicharan1 `checkMaxMin` should validate the min and max price of that specific ticket. In this I do not know which donation ticket (of a list of multiple) you are validating

Reply:
@niranjan94 I'm a little stuck in implementing this check for a specific donation ticket. How should I be doing this?

Reply:
@niranjan94 should be adding an` id` attribute to the div tag for min and max price, like here?
`div id="{{ticket.id}}_price"` & validate it by using `if (this.$('.ui.form').form('get value', '{{ticket.id}}_min_price')[0] < this.$('.ui.form').form('get value', '{{ticket.id}}_max_price')[0]) `

Reply:
But the problem here is that the ticket won't have an id till we add it. What do you suggest?